tests/bluetooth/tester/overlay-bt_ll_sw_split.conf

```diff
@@ -32,7 +32,6 @@ CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX=310
 
 # CSIP tests requires up to 3 devices
 CONFIG_BT_MAX_CONN=3
-CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT=3
 
 # BAP/BSRC/SCC/BV-38-C requires 2 BIGs
 CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
```

Comment:
Not a critical one; but depending on the test case this also might break MESH runs; because mesh can use up to 4 ext adv sets. Can we move this to Audio specific overlay?

Reply:
If this is meant to be used only for multicore chips I guess we can revert this in autopts.

However how is this than suppose to be configured for combined builds? Or it shouldn't ? ie LL configuration should be done based on host configs?

Reply:
If by combined builds you mean host and controller on same core, then for combined builds for mesh, using the `overlay-mesh.conf`, this is set to 32. It's not really a LL configuration per se, but a configuration for wherever the keys are stored I suppose

Applying the `overlay-mesh.conf` for a combined builds work, but it won't work if you attempt to only apply it for the host/appcore, nor will it work if you attempt to apply it for the controller/netcore on e.g. nRF5340